llm/sample/get_content.py

```diff
@@ -35,19 +36,40 @@
 
 
 # rss data
-rss_url='sample/files/xml/954689a5-3096-43a4-a80b-7810b219cef3.xml'
+rss_url = "sample/files/xml/954689a5-3096-43a4-a80b-7810b219cef3.xml"
 loader = UnstructuredXMLLoader(rss_url)
 xml_docs = loader.load()
 # print(xml_docs[0])
 
+
 # chunck
 splitter = RecursiveCharacterTextSplitter(
     chunk_size=2600,
     chunk_overlap=4,
-    separators=["\n\nEP", "\n\n", "\n", "(?<=\. )", " ", ""]
+    separators=["\n\nEP", "\n\n", "\n", "(?<=\. )", " ", ""],
 )
-xml_splitter = splitter.split_documents(xml_docs)
+xml_splits = splitter.split_documents(xml_docs)
 for i in range(1, 4):
-    print(xml_splitter[i])
-    print(xml_splitter[i].metadata)
+    print(xml_splits[i])
+    print(xml_splits[i].metadata)
     print()
+
+
+# embedding
+embeddings = OpenAIEmbeddings(model="text-embedding-3-large")
+
+
+# vectorstores
+""" once saved, there is no need to save again, as it may cause data duplication
+persist_directory = "sample/files/chroma/"
+vectordb = Chroma.from_documents(
+    documents=xml_splits, embedding=embeddings, persist_directory=persist_directory
+)
+vectordb.persist()
+"""
+
+question = "find contents related to ASML"
+docs = vectordb.similarity_search(question, k=3)
```

Comment:
I did a similarity_search example in _question_answering.py_  and the answer is so wrong as well